### PR TITLE
data: mansoor666.json

### PR DESCRIPTION
**Pull request by mansoor666**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/shaik.mb666.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "url": "http://shaik.mb666.io",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/shaik.mb666",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```
Comment (sital002) [inline data/shaik.mb666.json:17]:
```suggestion
    }
```
the trailing comma is not required

**Review by sital002 — CHANGES_REQUESTED** [review]
Your GitHub username must match your filename instead of ```data/shaik.mb666.json``` it should be ```data/mansoor666.json```
Please make the necessary changes to get your profile merged.

**Comment by mansoor666** [comment]
Updated file name

**Comment by mansoor666** [comment]
updated file name as required

**Comment by mansoor666** [comment]
updated file name as required.

**Comment by mansoor666** [comment]
m

**Review by accodes21 — COMMENTED** [review]
Ig you have already created your profile using form here: biodrop.io/mansoor666, you can close this PR as it will be obsolete

**Comment by SaraJaoude** [comment]
> Ig you have already created your profile using form here: biodrop.io/mansoor666, you can close this PR as it will be obsolete

I am closing this PR for the above reason.  

If you are looking to edit your existing Profile then you may want to check out our Editing Guide in [Docs.](https://www.biodrop.io/docs)

**Pull request closed without merging**